### PR TITLE
[GP5-16] Order - As a user, I want to select, add a new or change the address on the checkout page

### PR DESCRIPTION
**Pull request by RaihanAdity**

### CODE REVIEW

**Review by BarsMoxxi — APPROVED** [review]

File: projects/server/controller/cartController.js [hunk 1]
```diff
@@ -107,6 +108,213 @@ module.exports = {
...
+      console.log(cart);
+      res.send({ cart });
+    } catch (error) {
+      res.status(404).send({
+        isError: true,
+        message: error.message,
+        data: error,
+      });
+    }
+
```
Comment (doomsdoge) [inline projects/server/controller/cartController.js:316]: missing close bracket `},`

File: projects/server/controller/cartController.js [hunk 2]
```diff
@@ -107,6 +108,214 @@ module.exports = {
...
+    try {
+      const { id } = await db.user.findOne({
+        where: { uid: uid },
+      });
+
+      const cart = await db.cart.findAll({
+        where: { user_id: id },
+      });
+
+      console.log(cart);
```
Comment (haikal1223) [inline projects/server/controller/cartController.js:307]: unused code

File: projects/server/middleware/verifyToken.js [hunk 3]
```diff
@@ -1,27 +1,29 @@
...
-			res.status(401).send({
-				isError: true,
-				message: "Invalid Token",
-				data: null,
-			});
-		}
-	},
+    try {
+      const validateTokenResult = validateToken(token);
+      // console.log(validateTokenResult)
```
Comment (haikal1223) [inline projects/server/middleware/verifyToken.js:17]: unused code

**Review by haikal1223 — APPROVED** [review]

**Review by BarsMoxxi — APPROVED** [review]

File: projects/client/src/App.js [hunk 4]
```diff
@@ -119,42 +120,32 @@ function App() {
...
+            path="/order"
+            element={
+              <RequireAuth>
+                <Order login={isLoggedIn} />
+              </RequireAuth>
+            }
+          />
+          <Route path="/forgotpassword" element={<ForgotPassword />} />
+          <Route path="/updatePassword/:uid" element={<UpdatePassword />} />
+
```
Comment (doomsdoge) [inline projects/client/src/App.js:148]: keep admin routing

**Review by rohmatsetiawanmta — APPROVED** [review]

**Review by doomsdoge — APPROVED** [review]

**Pull request merged**